src/event-handler/receive.ts

```diff
@@ -1,4 +1,3 @@
-// @ts-ignore to address #245
```

Comment:
Please see #245 and #271

Reply:
Ah so these do need to remain as `@ts-ignore` - fair enough, I'll update.